template/helper/player.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @package    DOCman
+ * @copyright   Copyright (C) 2011 - 2014 Timble CVBA (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://www.joomlatools.com
+ */
+
+class ComFilesTemplateHelperPlayer extends KTemplateHelperAbstract
+{
+    protected static $HTML5_VIDEO_FORMATS = array('mp4', 'webm', 'ogg');
+
+    protected static $HTML5_AUDIO_FORMATS = array('aac', 'mp3', 'ogg', 'flac','x-flac', 'wave', 'wav', 'x-wav', 'x-pn-wav');
+
+    protected static $declared = false;
+
+    /*
+    public function load()
```

Comment:
@ercanozkaya in the FILElink content plugin I am replacing the urls with players and the plyr declarations is added to the header of the page only once. 

At first I ended up with 2 methods one using JDocument for the content plugin and another inserting <ktml:script> for the layouts. I had a complex logic to make sure the plyr declarations wouldn't happen twice. Then I decided to roll back to using a JDocument singleton, since I didn't know any KDocument way of inserting css and js declarations to the html header.